lib/src/main/kotlin/com/swmansion/starknet/data/types/Responses.kt

```diff
@@ -322,7 +328,7 @@ data class ProcessedBlockWithReceipts(
     override val status: BlockStatus,
 
     @SerialName("transactions")
-    override val transactionWithReceipts: List<TransactionWithReceipt>,
+    override val transactionsWithReceipts: List<TransactionWithReceipt>,
```

Comment:
nit: I see the point with differentiating between a list of transactions and transactions with receipts, but it might be easier for everyone if we just stick with calling it `transactions`.

Reply:
It seems more descriptive and we also we have `transactionHashes` field just below. I don't have a strong say on this though, @THenry14 WDYT?

Reply:
these are different types basically, transactions (hashes) and receipts. do not have a strong opinion as well, but I'd lean towards leaving this as is? it is indeed more descriptive